### PR TITLE
Add Unova dungeon Pokémon.ts

### PR DESCRIPTION
**Pull request by Ultima1990**
Stats based on Stark Mountain.

### CODE REVIEW

**Comment by anapple96** [comment]
Super unbalanced, top dungeons are easier than HoO, should be at least 3 - 4x the current figures

**Comment by Ultima1990** [comment]
I did say I'm not touching stats...

**Comment by anapple96** [comment]
On the Discord server?

File: src/scripts/dungeons/Dungeon.ts [hunk 1]
```diff
@@ -570,3 +570,274 @@ dungeonList['Stark Mountain'] = new Dungeon('Stark Mountain',
         new DungeonBossPokemon('Heatran', 9000000, 70),
     ],
     96500, 201, 35);
+
+// Unova
+
```
Comment (RedSparr0w) [inline src/scripts/dungeons/Dungeon.ts:575]:
```suggestion
// TODO: Balancing of dungeon Pokemon HP & rewards.
```

**Review by RedSparr0w — APPROVED** [review]
Nice work! 👍

**Pull request merged**